spec/acceptance/certs_spec.rb

```diff
@@ -21,11 +21,11 @@
       apply_manifest(pp, catch_failures: true)
     end
 
-    describe command("/usr/bin/curl https://#{fact('hostname')}.example.com:443") do
+    describe command("/usr/bin/curl https://#{fact('fqdn')}:443") do
```

Comment:
that's not related to the actual code changes, right? should we move it to a dedicated PR?

Reply:
I debated that, but consider it part of the "rewrite certs" PR. There I also switched to `$(hostname -f)`